### DISCUSSION TITLE
[Newb] How to get full libs to use this whisper-standalone-win completely offline?

### DISCUSSION THREAD

**Discussion by BanQuanChung** [Purfview/whisper-standalone-win#362] [Q&A] [answered]
Hello.
I'm brand new and know nothing about coding. Just looking for ways to transcribe mp3 files into text and I have found this to try.
(Main language: Vietnamese)

**1. Please show me the way to download all what is needed to use this completely offline.
2. Is it necessary to constantly keep the files up-to-date so that this machine works more and more precisely with transcribing?**

Many thanks to Purfview and any others contributing to this code.

**Comment by Purfview** [comment, marked as answer]
First, "whisper-standalone-win" is not some program to be used per se, it's just the repo name, so, you can just read it.

>I'm brand new and know nothing about coding. Just looking for ways to transcribe mp3 files into text and I have found this to try.

Looks like you found the right repo as there no coding involved to do what you want by using standalone executables.

>How to get full libs...

You don't need to get any "full libs", whatever that means.

>1. Please show me the way to download all what is needed to use this completely offline.

Just download a binary you want to use, then [auto]download models you need and it will be completely offline.
All that you can find on the main page.

>2. Is it necessary to constantly keep the files up-to-date so that this machine works more and more precisely with transcribing?

Not necessary, but updates are for something better, not worse.

**Reply by BanQuanChung** [reply]
I got it, Purf.

I have tried faster whisper with the following commands, 
`faster-whisper-xxl.exe %file_list% -pp -o source --batch_recursive --check_files --standard -f srt -m small --language Vietnamese --device cpu`
but the precision for the Vietnamese language transcription isn't as high as what I expect. And it took like 38min to finish for a 15min mp3 file. Quite a long time!
(Initially, I tried using auto captions function in CapCut Video Editing Soft. It's more precise and much quicker. But it requires internet, while I am looking for offline methods.)

**Could you please help me with enhancing the precision in Vietnamese transcribing using this faster whisper?**

**Reply by Purfview** [reply]
> the precision for the Vietnamese language transcription isn't as high as what I expect.

Of course it isn't with the small model, use `large-v2` model for the best precision.

> And it took like 38min to finish for a 15min mp3 file. Quite a long time!

Buy better computer with CUDA GPU.

PS
Please don't post here a nonsense generated by GPT.
